examples/customresources/models/mybase/mybase.go

```diff
@@ -50,6 +50,9 @@ func newBase(ctx context.Context, deps resource.Dependencies, conf resource.Conf
 func (b *myBase) Reconfigure(ctx context.Context, deps resource.Dependencies, conf resource.Config) error {
 	b.left = nil
 	b.right = nil
+
+	// A good practice is to use our helper function to access the converted attributes in the same
+	// form as the Config struct in this file
```

Comment:
Sorry to nitpick, but this says "it's a good practice to use this" but doesn't say why/what it does. At least it's confusing to me when I read it. Also, "what are "converted attributes"? That's not a concept on display.

I would think something more like "This takes the generic resource.Config passed down from the parent and converts it to the model specific (aka "native") Config structure defined above, making it easier use directly."

Reply:
no please nitpick away! I'm not the best at comments so feedback is super welcome.